src/sidebar/components/test/top-bar-test.js

```diff
@@ -18,9 +18,6 @@ describe('topBar', function() {
       })
       .component('searchInput', {
         bindings: require('../search-input').bindings,
-      })
```

Comment:
Note: There are no tests at all in `top-bar-test` for the `SortMenu`, which it renders. It passes no props. I don't know that there's anything _to_ test.